pycaching/cache.py

```diff
@@ -613,6 +616,20 @@ def description(self, description):
         description = str(description).strip()
         self._description = description
 
+    @property
+    @lazy_loaded
+    def description_html(self):
+        """The cache long description in raw HTML.
+
+        :type: :class:`str`
+        """
+        return self._description
```

Comment:
I just have  been confused because of the CI results, but this should be `return self._description_html` of course.

Reply:
Just fixed! Thanks for pointing that typo out!